javascript/code-challenges/challenges-14.test.js

```diff
@@ -0,0 +1,282 @@
+
+For example, ['apple', 'banana', 'MacGyver'] returns ['Apple', 'Banana', 'MacGyver'].
+------------------------------------------------------------------------------------------------ */
+
+const toTitleCase = (arr) => {
+  return arr.map(str => str[0].toUpperCase() + str.substr(1));
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named biggerThanLuke that, given the Star Wars data, below, returns the names of the characters whose mass is greater than Luke's.
+
+The names should be combined into a single string with each character name separated by a dash.
+
+For example, "Lando Calrisian - Boba Fett - Princess Amidala".
+------------------------------------------------------------------------------------------------ */
+
+let starWarsData = [{
+  name: 'Luke Skywalker',
+  height: '172',
+  mass: '77',
+  hair_color: 'blond',
+  skin_color: 'fair',
+  eye_color: 'blue',
+  birth_year: '19BBY',
+  gender: 'male',
+},
+{
+  name: 'C-3PO',
+  height: '167',
+  mass: '75',
+  hair_color: 'n/a',
+  skin_color: 'gold',
+  eye_color: 'yellow',
+  birth_year: '112BBY',
+  gender: 'n/a'
+},
+{
+  name: 'R2-D2',
+  height: '96',
+  mass: '32',
+  hair_color: 'n/a',
+  skin_color: 'white, blue',
+  eye_color: 'red',
+  birth_year: '33BBY',
+  gender: 'n/a'
+},
+{
+  name: 'Darth Vader',
+  height: '202',
+  mass: '136',
+  hair_color: 'none',
+  skin_color: 'white',
+  eye_color: 'yellow',
+  birth_year: '41.9BBY',
+  gender: 'male'
+},
+{
+  name: 'Leia Organa',
+  height: '150',
+  mass: '49',
+  hair_color: 'brown',
+  skin_color: 'light',
+  eye_color: 'brown',
+  birth_year: '19BBY',
+  gender: 'female'
+},
+{
+  name: 'Pex Kylar',
+  height: '180',
+  mass: '190',
+  hair_color: 'orange',
+  skin_color: 'brown',
+  eye_color: 'none',
+  birth_year: '27BBY',
+  gender: 'n/a'
+}];
+
+let biggerThanLuke = (arr) => {
+  const flag = arr.filter(item => item.name === 'Luke Skywalker');
+
+  let biggerThan = arr.filter(item => parseInt(item.mass) > parseInt(flag[0].mass));
+
+  let names = biggerThan.map(char => char.name);
+
+  return names.join(' - ');
+
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 4
+Write a function named sortBy that takes in an array of objects, each of which has a particular property, and sorts those objects by that property, lowest to highest, returning the same array.
+
+Here is an example of the input:
+[
+  {name: 'Sweatshirt', price: 45},
+  {name: 'Bookmark', price: 2.50},
+  {name: 'Tote bag', price: 15}
+];
+
+This data could be sorted by name or price.
+------------------------------------------------------------------------------------------------ */
+
+const sortBy = (property, arr) => {
+
+  return arr.sort((a, b) => a[property] - b[property]);
+
+};
```

Comment:
Hi there!

Nice work so far on these 👍 

Feedback here: Maybe you can say If a[property] > b[property]
   return something...
ElSE if a[property] < b[property] return something...

Review sort() method and I bet you will find the solution :)